launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -229,6 +233,49 @@ QSet<QString> MinecraftInstance::traits() const
     return profile->getTraits();
 }
 
+void MinecraftInstance::populateLaunchMenu(QMenu* menu)
```

Comment:
UI code technically doesn't belong here. But a FIXME should be fine enough :D


```suggestion
// FIXME: move UI code out of MinecraftInstance
void MinecraftInstance::populateLaunchMenu(QMenu* menu)
```

Reply:
Where should it be put in future in your opinion?